### PR TITLE
Update chia.toml with 96 missing repos

### PR DESCRIPTION
**Pull request by mando222**
Added in a lot of missing repos to the chia.toml.

### CODE REVIEW

File: data/ecosystems/c/chia.toml [hunk 1]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/gowke/lotus-blockchain"
+
+[[repo]]
+ url ="https://github.com/KryptomineCH/CHIA-RPC"
+
+[[repo]]
+ url ="https://github.com/L4bb3rs/Chia-Py-RPC"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-marketplace-admin-dapp"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2096]: Can remove

File: data/ecosystems/c/chia.toml [hunk 2]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+
+[[repo]]
+ url ="https://github.com/FireAcademy/telemetry"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2099]: how's this using Chia?

File: data/ecosystems/c/chia.toml [hunk 3]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+
+[[repo]]
+ url ="https://github.com/bramcohen/chia-gaming"
+
+[[repo]]
+ url ="https://github.com/FireAcademy/prices"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2105]: This one can be removed

File: data/ecosystems/c/chia.toml [hunk 4]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+
+[[repo]]
+ url ="https://github.com/lourkeur/.chia.nix.old"
+
+[[repo]]
+ url ="https://github.com/lmacken/plow"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2111]: Please remove

File: data/ecosystems/c/chia.toml [hunk 5]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+
+[[repo]]
+ url ="https://github.com/ydipeepo/chia-k8s"
+
+[[repo]]
+ url ="https://github.com/MarcusL11/chamster"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2117]: Remove please

File: data/ecosystems/c/chia.toml [hunk 6]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/SlowestTimelord/nftsonchia"
+
+[[repo]]
+ url ="https://github.com/tjallingk/ChiaSyncBench"
+
+[[repo]]
+ url ="https://github.com/mpwsh/chia-models-rs"
+
+[[repo]]
+ url ="https://github.com/konstantin-v03/trade_bot"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2144]: can be removed

File: data/ecosystems/c/chia.toml [hunk 7]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/mx-pool/super-farmer"
+
+[[repo]]
+ url ="https://github.com/0xgorro/chia-token-gated-access"
+
+[[repo]]
+ url ="https://github.com/Yakuhito/tibet-analytics"
+
+[[repo]]
+ url ="https://github.com/iced-caffeine/py_dexie_tools"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2261]: Can remove

File: data/ecosystems/c/chia.toml [hunk 8]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/asicminer8/plotctl"
+
+[[repo]]
+ url ="https://github.com/inan0812/goji-blockchain"
+
+[[repo]]
+ url ="https://github.com/Fluffy9/Instaplot"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/icons"
```
Comment (eherrerosj) [inline data/ecosystems/c/chia.toml:2183]: Can remove

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Hi @mando222, thanks for the PR on Chia. I've left a few change requests mainly because some of the repos don't satisfy the requirements to be added to the Chia ecosystem. Please remove the ones I pointed out so we can merge the rest of the changes

File: data/ecosystems/c/chia.toml (hunk 2 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2099]: FireAcademy is an infra as a service provider for our community.  My understanding was that this project was a data gathering project in that context.

File: data/ecosystems/c/chia.toml (hunk 4 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2111]: Was this a duplicate?  It is a chia plot mover.

File: data/ecosystems/c/chia.toml (hunk 7 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2261]: This is a tool to aggregate all the chia offer files on the dexie service.  It has everything to do with the Chia Network.

File: data/ecosystems/c/chia.toml [hunk 9]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/keyspaceapp/swift-vdf"
+
+[[repo]]
+ url ="https://github.com/mindspice/JChia-RPC-Library"
+
+[[repo]]
+ url ="https://github.com/keyspaceapp/swift-clvm-tools"
+
+[[repo]]
+ url ="https://github.com/AngelHub/save_the_kittens_part_2"
```
[the remark this answers is not in the document]
Comment (mando222) [reply data/ecosystems/c/chia.toml:2327]: You are correct about line 2327.  That one slipped through.  Will remove it.

File: data/ecosystems/c/chia.toml [hunk 10]
```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
...
+ url ="https://github.com/KryptomineCH/Chia-NFT-Minter"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/api"
+
+[[repo]]
+ url ="https://github.com/CryptoLFDM/ChiaRPCWrapper"
+
+[[repo]]
+ url ="https://github.com/Chia-Network/cadt-ui"
```
[the remark this answers is not in the document]
Comment (mando222) [reply data/ecosystems/c/chia.toml:2081]: Will remove

File: data/ecosystems/c/chia.toml (hunk 5 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2117]: Will do.  Just heads up this one might have more added to it in the future.

File: data/ecosystems/c/chia.toml (hunk 6 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2144]: will do

File: data/ecosystems/c/chia.toml (hunk 8 above)
Comment (mando222) [reply data/ecosystems/c/chia.toml:2183]: will do

**Comment by mando222** [comment]
@eherrerosj Would love a second look at things.  I can make more changes if needed but really want to get this in as soon as possible.

File: data/ecosystems/c/chia.toml (hunk 4 above)
Comment (eherrerosj) [reply data/ecosystems/c/chia.toml:2105]: my bad, let's keep it

File: data/ecosystems/c/chia.toml (hunk 7 above)
Comment (eherrerosj) [reply data/ecosystems/c/chia.toml:2246]: in general we tend to only add stuff that's built on top of ecosystems. Let's just keep it given that the repo has the chia tag on it

**Review by eherrerosj — APPROVED** [review]
thanks for the contribution @mando222, merging now

**Pull request merged**